.github/workflows/documentation-preview.yml

```diff
@@ -0,0 +1,47 @@
+name: Documentation Preview Generator
+
+on:
+  workflow_run:
+    workflows: [Documentation Preview Request]
+    types:
+      - completed
+
+env:
+  PREVIEW_PUBLISH_USERNAME: ${{ secrets.PREVIEW_PUBLISH_USERNAME }}
+  PREVIEW_PUBLISH_REPOSITORY: ${{ secrets.PREVIEW_PUBLISH_REPOSITORY }}
+  PREVIEW_PUBLISH_BRANCH: ${{ secrets.PREVIEW_PUBLISH_BRANCH }}
+  PREVIEW_PUBLISH_TOKEN: ${{ secrets.PREVIEW_PUBLISH_TOKEN }}
+
+jobs:
+  documentation-preview:
+    runs-on: ubuntu-latest
```

Comment:
May recall to add a condition on trigger completed + is successfully??

Reply:
Sorry I had forgotten to add the condition for successful completion. I've added it.